### PR TITLE
[Docs] Adds pages for configuring the systemd services run by MicroK8s

### PR DESCRIPTION
**Pull request by evilnick**
This adds some detail on the services running, with (updated) links to component docs

### CODE REVIEW

**Comment by evilnick** [comment]
Hi @ktsakalozos @tvansteenburgh, I have a question about this page [the demo render is here](https://microk8s-io-canonical-web-and-design-pr-140.run.demo.haus/docs/configuring-services).

Do we want to include the defaults? On the pro side, it provides some handy reference for how everything is set up and some of the things you might want to change. On the con side, it will make the page much much longer (I haven't included all of them in this PR at present), and some of the values aren't really defaults as they are generated at install time (e.g IP addresses). Any thoughts?

**Comment by ktsakalozos** [comment]
> Do we want to include the defaults?

I would -1 this, especially since each channel has its own defaults. What we could do is mention that the default values for each service can be found at `${SNAP}/default-args/`.

File: docs/configuring-services.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+### snap.microk8s.daemon-apiserver
+
+The Kubernetes API server validates and configures data for the API objects
+which include pods, services, replication controllers, and others. The API
+Server services REST operations and provides the frontend to the cluster’s
+shared state through which all other components interact.
+
+The service configuration is described in full in the upstream  
+[kube-apiserver documentation][kube-apiserver].
+
```
Comment (ktsakalozos) [inline docs/configuring-services.md]: I would add here where the configuration file is as we do for the rest of the services below.

File: docs/configuring-services.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+**Default configuration**:
```
Comment (ktsakalozos) [inline docs/configuring-services.md]: As mentioned, I am -1 to this.

File: docs/configuring-services.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+specifications, data locality, inter-workload interference, deadlines, and so
+on. Workload-specific requirements will be exposed through the API as
+necessary.
+
+The kube-scheduler daemon started using the arguments in
+`${SNAP_DATA}/args/kube-scheduler`. These are explained fully in the
+upstream [kube-scheduler documentation][kube-scheduler].
+
+
+To reconfigure a service you will need to edit the corresponding file and then restart the respective daemon. For example:
```
Comment (ktsakalozos) [inline docs/configuring-services.md]: This is mentioned at the very top.

File: docs/configuring-services.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+
```
Comment (ktsakalozos) [inline docs/configuring-services.md]: I would also add flannel here.

Comment (evilnick) [reply docs/configuring-services.md]: I added a flannel section in the appropriate place, please check it out!

File: docs/configuring-services.md (hunk 1 above)
Comment (evilnick) [reply docs/configuring-services.md]: done

**Comment by evilnick** [comment]
I swear 30% of the time the 'test-site' check fails for fun

**Pull request merged**